### PR TITLE
Update versions, add msg to AdyenError, fix console

### PR DESCRIPTION
**Pull request by crrood**
Pull requests 22, 23, 24

### CODE REVIEW

File: lib/adyen/services/payments.rb [hunk 1]
```diff
@@ -3,7 +3,7 @@
 module Adyen
   class Payments < Service
     attr_accessor :version
-    DEFAULT_VERSION = 40
+    DEFAULT_VERSION = 46
```
Comment (Aleffio) [inline lib/adyen/services/payments.rb]: Can we change PAL to v49 like in other libraries or do we need additional steps?
https://github.com/Adyen/adyen-java-api-library/blob/d70c84a478be3650b2afbc897edfdbe449aba005/src/main/java/com/adyen/Client.java#L38

**Comment by rikterbeek** [comment]
@crrood as payment methods will break with v49 we need to create a mayor release for this so 3.0.0. Let's update all the endpoints to there latest version. can you update ?

File: lib/adyen/version.rb [hunk 2]
```diff
@@ -1,4 +1,4 @@
 module Adyen
   NAME = "adyen-ruby-api-library"
-  VERSION = "2.0.0".freeze
+  VERSION = "2.0.1".freeze
```
Comment (rikterbeek) [inline lib/adyen/version.rb]: @crrood as payment methods will break with v49 we need to create a mayor release for this so 3.0.0. Let's update all the endpoints to there latest version. can you update ?

File: lib/adyen/errors.rb [hunk 3]
```diff
@@ -1,6 +1,6 @@
 module Adyen
   class AdyenError < StandardError
-    attr_reader :code, :response, :request
+    attr_reader :code, :response, :request, :msg
```
Comment (x4d3) [inline lib/adyen/errors.rb]: could we have a `message` method containing a summary of all the properties ?
```
    def message
      attributes = {code: code, response: response, request: request, msg: msg}.compact.map{ |k, v| "#{k}:#{v}"}.join(", ")
      "#{self.class.name} #{attributes}"
    end
```
and 
```
Adyen::AuthenticationError.new("message", "request").message
```
would render
```
"Adyen::AuthenticationError code:401, request:request, msg:message"
```
which would definitely be helpful

Comment (crrood) [reply lib/adyen/errors.rb]: Hi Xavier,

Sure, I have no problem with adding a convenience method to access these properties.  Can you submit a pull request?

Comment (x4d3) [reply lib/adyen/errors.rb]: Thanks @crrood .

I've created the PR: https://github.com/Adyen/adyen-ruby-api-library/pull/27

**Pull request merged**